lib/phx_tools_web/components/shared_compoents.ex

```diff
@@ -0,0 +1,72 @@
+defmodule PhxToolsWeb.Components.SharedCompoents do
+  @moduledoc """
+  Renders optimum logo
+  """
+  import Phoenix.Component
```

Comment:
That is not how the components are made.
```suggestion
  use PhxToolsWeb, :component
```

Reply:
I've made a mistake. In Phoenix 1.7 it's `use PhxToolsWeb, :html`.